lib/js/components/Banner/Banner.vue

```diff
@@ -0,0 +1,375 @@
+<template>
+	<div class="o-banner" :class="[colorClass, layoutClass]">
+		<div class="o-banner__content">
+			<div class="o-banner__header">
+				<div v-if="icon" class="o-banner__iconContainer">
+					<!-- TODO: https://bethink.atlassian.net/browse/IT-3589 change to a-illustration in the future -->
+					<ds-icon class="o-banner__icon" :class="[colorClass]" :icon="icon" />
+				</div>
+
+				<div class="o-banner__text">
+					<div class="o-banner__title" v-text="title" />
+					<div v-if="hasDefaultText" class="o-banner__defaultText">
+						<slot name="defaultText" />
+					</div>
```

Comment:
`defaultText` is a bit misleading IMO. I thought it's some kind of a default that can be overriden. In Figma they used "basic" word, maybe `basicText` would be clearer?

Reply:
Slot names are defined in miro https://miro.com/app/board/uXjVOZ_Oix8=/?moveToWidget=3458764527823093441&cot=10